pkg/bloomgateway/bloomgateway_test.go

```diff
@@ -216,6 +216,12 @@ func TestBloomGateway_FilterChunkRefs(t *testing.T) {
 	})
 
 	t.Run("request cancellation does not result in channel locking", func(t *testing.T) {
+		t.Skip(`
```

Comment:
Shall we add a TODO here so we don't skip it in the future?

Reply:
I would rather remove the test completely